src/sqlancer/mysql/ast/MySQLSelect.java

```diff
@@ -18,6 +18,10 @@ public void setSelectType(SelectType fromOptions) {
         this.setFromOptions(fromOptions);
     }
 
+    public SelectType getSelectType() {
```

Comment:
Isn't this duplicate to the method below?

Reply:
Aha yes. Will fix later.